contracts/interfaces/IPooledStaking.sol

```diff
@@ -0,0 +1,15 @@
+
+interface IPooledStaking {
+    function pushReward(address contractAddress, uint amount) external;
+    function pushBurn(address contractAddress, uint amount) external;
+
+    function contractStakedAmount(address contractAddress) external view returns (uint);
+    function stakerReward(address staker) external view returns (uint);
```

Comment:
these 2 getters don't exist yet @shark0der  need smth like this or any other name you see a good fit